integr8ly/cloud-resource-operator
# add basic standalone vpc peering logic

add logic to the standalone network provider for peering a provided
vpc to the cluster vpc.

this is intended to be used to peer the vpc of the standalone cloud
resources vpc to the cluster to allow for database and cache
connectivity from services running within the openshift cluster.

verification:
- replace the 'placeholder' id of the vpc returned in
  'CreateNetwork' in cluster_network_provider.go with the id of
  a real vpc in aws. this should not be the cluster id.
- run the operator against a cluster running in the same aws
  account as the vpc provided.
- create a postgres or redis custom resource.
- ensure there is a vpc peering connection created in the aws
  account between the two vpcs and that it's state is active
- ensure there are tags on the peering connection.
